postapi/Controllers/AddPostController.cs

```diff
@@ -9,5 +16,104 @@ namespace postapi.Controllers
 
     public class AddPostController : ControllerBase
     {
+
+        private readonly IConfiguration _configuration;
+        public AddPostController(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+        [Route("GetPosts")]
+        [HttpGet]
+        public JsonResult GetPosts()
```

Comment:
use try and catch block in every action.
use return type IActtionResult to handle multiple return type of action method

Reply:
Updated the add post